features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -186,6 +186,8 @@
         <RenewRefreshTokenForRefreshGrant>true</RenewRefreshTokenForRefreshGrant>
         <!-- Process the token before storing it in database, e.g. encrypting -->
         <TokenPersistenceProcessor>org.wso2.carbon.identity.oauth.tokenprocessor.PlainTextPersistenceProcessor</TokenPersistenceProcessor>
+        <!-- If the user is a federated, user will not be able to access claims from local userstore even if the username matches -->
+        <mapFederatedUsersToLocal>false</mapFederatedUsersToLocal>
```

Comment:
Better to avoid camel case, since all other configurations use capital letter to start the tag name. So rename to,
MapFederatedUsersToLocal

Reply:
Fixed them